### PR TITLE
github ci/non bundled libhtp/v2

### PR DESCRIPTION
**Pull request by jasonish**
- **github-ci/builds: don't run if only doc changes**
- **github-ci: add non-bundled libhtp build**

### CODE REVIEW

**Comment by jasonish** [comment]
Draft as I need to make this a dummy master somewhere to make sure all the variables make it through on a pull request, but likely good as-is.

File: .github/workflows/build-centos-7.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@ name: build-centos-7
 
 on:
   push:
+    paths-ignore:
+      # Don't run this workflow if only files under doc/ have been
+      # modified.
+      - "doc/**"
```
Comment (jasonish) [inline .github/workflows/build-centos-7.yml:8]: Should be applied to the other workflows as well.

Comment (victorjulien) [reply .github/workflows/build-centos-7.yml:8]: with exception of doc builders :)

**Comment by jasonish** [comment]
Replaced by https://github.com/OISF/suricata/pull/11407.

**Pull request closed without merging**